src/gui/editors/PianoRoll.cpp

```diff
@@ -4590,6 +4590,43 @@ int PianoRoll::quantization() const
 	return DefaultTicksPerBar / Quantizations[m_quantizeModel.value() - 1];
 }
 
+void PianoRoll::strumNotesDn() //down-top
+//Method for chord-strumming.
+//The strum-distance is the chosen Q-value
+{
+  if( ! hasValidPattern() )
```

Comment:
`hasValidPattern` has been renamed to `hasValidMidiClip`

Reply:
Ok i will fix that

Reply:
Fixed